### PR TITLE
refactor: use semi-transparent background for striped rows

### LINKED ISSUE #195: Table stripe color should be semi-transparent
This doesn't look right:

image

I'm also trying to allow users to tweak the site's background color: https://github.com/facebook/docusaurus/pull/5822 and any other color doesn't look right with the table stripes still staying dull-black.

### PR DESCRIPTION
**Pull request by lex111**
Fixes #195

@Josh-Cena take a look please.

### CODE REVIEW

**Review by Josh-Cena — APPROVED** [review]
That looks nice! I wonder if it can be slightly _less_ transparent so as to retain contrast in dark mode, but it looks much better

**Comment by lex111** [comment]
Like this?

| Before   | After    |
| -------- | -------- |
| image |  image |

**Comment by Josh-Cena** [comment]
Yes, that looks good 👍

**Comment by slorber** [comment]
sorry missed this PR since I don't use email notifications anymore 😅 

That looks fine 👍

File: packages/core/styles/common/dark-mode.pcss [hunk 1]
```diff
@@ -38,6 +38,8 @@ html[data-theme='dark'] {
   --ifm-scrollbar-thumb-background-color: #686868;
   --ifm-scrollbar-thumb-hover-background-color: #7a7a7a;
 
+  --ifm-table-stripe-background: rgba(255, 255, 255, 0.07);
```
Comment (slorber) [inline packages/core/styles/common/dark-mode.pcss:41]: Was wondering if we ever used this `default-dark/default-dark.pcss` file @lex111 @yangshun ?

https://deploy-preview-204--infima.netlify.app/demo/css/default-dark/default-dark.css

As far as I understand it's not used by Docusaurus and it's not either really maintained (for example current PR does not update it, and probably many others too)

Maybe we could simply delete this entrypoint?

Comment (slorber) [reply packages/core/styles/common/dark-mode.pcss:41]: Docusaurus is using this file only:

```js
function getInfimaCSSFile(direction: string) {
  return `infima/dist/css/default/default${
    direction === 'rtl' ? '-rtl' : ''
  }.css`;
}
```

**Pull request merged**